JavaAssignments/src/saket/swapwiththirdvaiable.java

```diff
@@ -0,0 +1,24 @@
+package saket;
+
+public class swapwiththirdvaiable {
+
+	int a=10;
```

Comment:
Why you need instance variable. as you are using these variables in method only. it should be local variable if usage of variable is limited to method. I suggest use parameters of method .. dont use such hardcode value.